### PR TITLE
filter feature of squinote

### PR DESCRIPTION
**Pull request by devAmlan**
Filter  Features in the Note app.

1. Users can search notes on note titles.
2. Users can Filter notes according to the Label.

AnyConv com__squinote-filter

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
Clean UI design and code. Would suggest adding comments explaining Filter() for future reference. Good to merge

File: frontend/src/component/filter/Filter.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.filter {
+  width: 300px;
```
Comment (humzahasan) [inline frontend/src/component/filter/Filter.css:2]: Use rem here

File: frontend/src/component/filter/Filter.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.filter {
```
Comment (humzahasan) [inline frontend/src/component/filter/Filter.css:1]: use rems instead of px while refactoring

**Review by humzahasan — APPROVED** [review]
- UI looks great
- Good folder structure
- Clean and readable code

Good to merge!

File: frontend/src/component/filter/Filter.css (hunk 2 above)
Comment (devAmlan) [reply frontend/src/component/filter/Filter.css:1]: I will change it to rem, thanks @humzahasan .

**Pull request merged**